### PR TITLE
Fix nuget package creation

### PR DESCRIPTION
**Pull request by gingters**

### CODE REVIEW

File: .github/workflows/build-and-publish-on-push.yaml [hunk 1]
```diff
@@ -17,34 +16,36 @@ jobs:
...
+          DOTNET_NOLOGO : true
+          DOTNET_CLI_TELEMETRY_OPTOUT : true
 
       - name: Restore required NuGet packages
         run: dotnet restore
         working-directory: src
 
       - name: Build
-        run: dotnet build -c:Release -p:ContinuousIntegrationBuild=true -p:PreReleaseSuffix=.$GITHUB_RUN_NUMBER
+        run: dotnet build -c:Release -p:ContinuousIntegrationBuild=true -p:PreReleaseSuffix=-$GITHUB_RUN_NUMBER
```
Comment (thomashilzendegen) [inline .github/workflows/build-and-publish-on-push.yaml:31]: Maybe adding `--no-restore` could speed this up?

File: docs/features.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# RelayServer features
```
Comment (thomashilzendegen) [inline docs/features.md:1]: Maybe this slipped in?

Comment (gingters) [reply docs/features.md:1]: indeed

File: .github/workflows/build-and-publish-on-push.yaml (hunk 1 above)
Comment (gingters) [reply .github/workflows/build-and-publish-on-push.yaml:31]:
```suggestion
        run: dotnet build -c:Release -p:ContinuousIntegrationBuild=true -p:PreReleaseSuffix=-$GITHUB_RUN_NUMBER --no-restore
```

Comment (gingters) [reply .github/workflows/build-and-publish-on-push.yaml]: Probably not really, but it can't hurt ;)

File: docs/features.md (hunk 2 above)
Comment (gingters) [reply docs/features.md]: unwhoopsed. --fixup ftw :)

**Review by thomashilzendegen — APPROVED** [review]

**Pull request merged**